<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabTokens">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/font.png</normaloff>:/icons/font.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Токены</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableTokens">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>30</number>
            </attribute>
            <column>
             <property name="text">
              <string>Лексема</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Строка</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Колонка</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Позиция</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Длина</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Класс</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabTree">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/chart_organisation.png</normaloff>:/icons/chart_organisation.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Дерево</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <column>
             <property name="text">
              <string notr="true">Пусто</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/icons/page_white_code.png</normaloff>:/icons/page_white_code.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Исходный код</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="textEditCode">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget_3">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabErrors">
        <attribute name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Ошибки</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEditError">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabStack">
        <attribute name="icon">
         <iconset>
          <normaloff>database.png</normaloff>
          <normalon>:/icons/database.png</normalon>database.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Стек</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menuExamples">
     <property name="title">
      <string>Примеры</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/icons/page_white_stack.png</normaloff>:/icons/page_white_stack.png</iconset>
     </property>
     <addaction name="actionHighlighter"/>
     <addaction name="actionHighlighterH"/>
     <addaction name="actionMain"/>
     <addaction name="actionMainwindow"/>
     <addaction name="actionMainwindowH"/>
     <addaction name="actionToken"/>
     <addaction name="actionTokenH"/>
     <addaction name="actionTokenizer"/>
     <addaction name="actionTokenizerH"/>
     <addaction name="separator"/>
     <addaction name="actionlex_c"/>
     <addaction name="actionexpr_c"/>
     <addaction name="separator"/>
     <addaction name="actionqaxserverbase_cpp"/>
     <addaction name="separator"/>
     <addaction name="actionerror_str_cpp"/>
     <addaction name="actionerror_comm_cpp"/>
     <addaction name="separator"/>
     <addaction name="actiondiff_literals_cpp"/>
    </widget>
    <addaction name="actionOpenFile"/>
    <addaction name="menuExamples"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/folder_page.png</normaloff>:/icons/folder_page.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionHighlighter">
   <property name="text">
    <string>highlighter.cpp</string>
   </property>
  </action>
  <action name="actionHighlighterH">
   <property name="text">
    <string>highlighter.h</string>
   </property>
  </action>
  <action name="actionMain">
   <property name="text">
    <string>main.cpp</string>
   </property>
  </action>
  <action name="actionMainwindow">
   <property name="text">
    <string>mainwindow.cpp</string>
   </property>
  </action>
  <action name="actionMainwindowH">
   <property name="text">
    <string>mainwindow.h</string>
   </property>
  </action>
  <action name="actionToken">
   <property name="text">
    <string>token.cpp</string>
   </property>
  </action>
  <action name="actionTokenH">
   <property name="text">
    <string>token.h</string>
   </property>
  </action>
  <action name="actionTokenizer">
   <property name="text">
    <string>tokenizer.cpp</string>
   </property>
  </action>
  <action name="actionTokenizerH">
   <property name="text">
    <string>tokenizer.h</string>
   </property>
  </action>
  <action name="actionlex_c">
   <property name="text">
    <string>lex.c</string>
   </property>
  </action>
  <action name="actionexpr_c">
   <property name="text">
    <string>exp.c</string>
   </property>
  </action>
  <action name="actionqaxserverbase_cpp">
   <property name="text">
    <string>qaxserverbase.cpp</string>
   </property>
  </action>
  <action name="actionerror_str_cpp">
   <property name="text">
    <string>error_str.cpp</string>
   </property>
  </action>
  <action name="actionerror_comm_cpp">
   <property name="text">
    <string>error_comm.cpp</string>
   </property>
  </action>
  <action name="actiondiff_literals_cpp">
   <property name="text">
    <string>diff_literals.cpp</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
